package.json

```
@@ -2844,6 +2844,253 @@
+													},
+													{
+														"key": "I",
+														"name": "Find symbol in project",
+														"type": "command",
+														"command": "workbench.action.showAllSymbols"
+													},
+													{
+														"key": "R",
+														"name": "Find all references",
+														"type": "command",
+														"command": "references-view.findReferences"
+													}
+												]
+											},
+											{
+												"key": "r",
+												"name": "+Refactor",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "a",
+														"name": "Mark function as async",
+														"type": "command",
+														"command": "cmstead.jsRefactor.markAsAsync"
+													},
+													{
+														"key": "e",
+														"name": "Add export",
+														"type": "command",
+														"command": "cmstead.jsRefactor.addExport"
+													},
+													{
+														"name": "Negate expression",
+														"type": "command",
+														"command": "cmstead.jsRefactor.negateExpression"
+													},
+													{
+														"key": "p",
+														"name": "Shift params",
+														"type": "command",
+														"command": "cmstead.jsRefactor.shiftParams"
+													},
+													{
+														"key": "r",
+														"name": "Rename",
+														"type": "command",
+														"command": "cmstead.jsRefactor.rename"
+													},
+													{
+														"key": "s",
+														"name": "select from all refactorings",
+														"type": "command",
+														"command": "cmstead.jsRefactor.selectRefactoring"
+													},
+													{
+														"key": "c",
+														"name": "+Convert",
+														"type": "bindings",
+														"bindings": [
+															{
+																"key": ">",
+																"name": "Convert function to arrow function",
+																"type": "command",
+																"command": "cmstead.jsRefactor.convertToArrowFunction"
+															},
+															{
+																"key": "d",
+																"name": "Convert function expression to function declaration",
+																"type": "command",
+																"command": "cmstead.jsRefactor.convertToFunctionDeclaration"
+															},
+															{
+																"key": "e",
+																"name": "Convert arrow function to function expression",
+																"type": "command",
+																"command": "cmstead.jsRefactor.convertToFunctionExpression"
+															},
+															{
+																"key": "t",
+																"name": "Convert string concat to template literal",
+																"type": "command",
+																"command": "cmstead.jsRefactor.convertToTemplateLiteral"
+															}
+														]
+													},
+													{
+														"key": "w",
```

```suggestion
														"key": "s",
```
Should the key be `s` then?

---

I ended up dropping this, [unfortunately](https://github.com/VSpaceCode/vspacecode.github.io/pull/43).